### ISSUE TITLE
Unable to create Sales Order for customer

### ISSUE THREAD

**Issue by Guusggg** [inventree/InvenTree#4830]
Hi,

Today I wanted to create a Sales Order by going to the Customer in the Sell dropdown. When I found the customer and clicked on `New Sales Order`, nothing happened. When I create a Sell Order by not going to a specific customer, but on the Sell Order page, a form pops up where I can create the order. I expected the same with `New Sales Order` on a specific customer.

Is this a bug or am I doing something wrong?

```
# Version Information:
InvenTree-Version: 0.11.0
Django Version: 3.2.18
Commit Hash: 593a716
Commit Date: 2023-04-18
Database: postgresql
Debug-Mode: False
Deployed using Docker: True
Active plugins: False
```

**Comment by SchrodingersGat** [comment]
Closed in https://github.com/inventree/InvenTree/pull/4782

@Guusggg this has been fixed already, will be back-ported to 0.11.1 but otherwise is fixed in master

For now I'd suggest you create the new order from the "Sales Order" base page

**Comment by Guusggg** [comment]
Thank you very much, sorry for not looking into the patch notes!
